API.md

```diff
@@ -2106,6 +2106,7 @@ following options:
         - `'error'` - return a Bad Request (400) error response. This is the default value.
         - `'log'` - report the error but continue processing the request.
         - `'ignore'` - take no action and continue processing the request.
+        - `function` - takes a callback with optional params '(request, reply, source, error)'.
```

Comment:
I wouldn't call is optional params.

- a function with signature `function(request, reply, source, err)` where:
   - `request` - ...

and so on. See other function options style in the doc.